BUILDING.md

```diff
@@ -1,11 +1,51 @@
+# Build-System CLI
+
+## Non-interactive
+```
+python build.py -h, --help
+
+usage: build.py [-h] --target {android,linux,macos,win32} --arch {x86,x64,arm}
+                [--vendor VENDOR] [--keep-native-libs] [--node-enabled]
+                [--docker] [--vagrant] [--sys-image SYS_IMAGE] [--no-shutdown]
+                [--interactive]
+                [build-steps [build-steps ...]]
+```
+```
+python build.py -v alpine -t linux -a x64 -dkr -img openjdk:8u131-alpine -ne j2v8
+```
+
+## Interactive
+```
+python build.py --i, --interactive
+
+entering interactive mode...
+
+[0] Docker >> android-x86 >> NODE_ENABLED
+[1] Docker >> android-arm >> NODE_ENABLED
+[2] Docker >> alpine-linux-x64 >> NODE_ENABLED
+[3] Docker >> linux-x64 >> NODE_ENABLED
```

Comment:
> Why did you choose these ones as suggestions?
> I though interactive would be that I would pick target, vendor, platform, etc, interactively.. 

Because not all combinations of `platforms * architectures * vendors * virtualization-tech * feature-variations[N]` are currently technically possible nor would it be sensible to try and support them all.

For the current suggestions I added the ones that are working and that were important for me to test the cross-platform build.

> Is it possible to add suggestions?

Yes, have a look at [build_system/build_configs.py](https://github.com/eclipsesource/J2V8/pull/327/files#diff-ebf9f4d48dab1eb643ae630bc21d59db)

> also I found a tiny bug, it is -i or --interactive, --i doesn't work

You can use `-i` or `--interactive` ... they are both doing the same thing (they are aliases)